### PR TITLE
Handle CWL Success Code and Permanent Fail Code

### PR DESCRIPTION
**Pull request by anisahn09**
# Background
There are three exit codes that are handled specifically in CWL:
- `successCode` = Exit codes that indicate the process completed successfully.
- `temporaryFailCode` = Exit codes that indicate the process failed due to a **temporary** condition, will retry the task.
- `permanentFailCode` = Exit codes that indicate the process failed due to a **permanent** logic error, will stop the task.

For now, we will only handle `successCode` and `permanentFailCode` because `temporaryFailCode` need more complex adjustment on celery code to create the retry mechanism.

# Changes
Since `successCode` behaviour is similar with existing `custom_exit_behaviour` == CONTINUE and `permanentFailCode` behaviour is similar with existing `custom_exit_behaviour` == TERMINATE, I utilise these logic.

New field:
- `custom_success_exit` = this field will store the `successCode` from CWL
- `custom_terminate_exit` = this field will store the `permanentFailCode` from CWL
- `custom_fail_exit` = this field will NOT store anything from CWL, just for manual edit from admin frontend

Deleted field:
- `custom_exit_status`
- `custom_exit_behaviour`

Basically, data stored in these two field were copied to one of the new fields. For example:
`hspred_checkchain` task initially has `custom_exit_status` = 70,70 and `custom_exit_behaviour` = 1 (TERMINATE)

With these changes, `hspred_checkchain` task will have `custom_terminate_exit` = 70,70

# Note
I have also created the migration script to copy existing `custom_exit_status` to the new field. Kindly need your help to try if it also work in your local

### CODE REVIEW

**Comment by waleedev01** [comment]
just want to make sure, we do not have anything custom now in the database or cwl file?  @anisahn09

**Comment by anisahn09** [comment]
> just want to make sure, we do not have anything custom now in the database or cwl file? @anisahn09

yes, we do not handle any custom field from cwl
